### PR TITLE
TS-1769 local e2e eligible single over 55

### PR DESCRIPTION
**Pull request by LBHTKarki**
Local e2e tests for the following scenarios:
1.  Add household members to the household at the beginning of the application
2. Eligible main applicant only (over 55 years old)
3. Eligible application with household members

Also fixes the sample test flow.

### CODE REVIEW

**Review by martapederiva — APPROVED** [review]
🚢

File: cypress/pages/apply/[resident]/your-situation.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ class ApplyResidentYourSituationPage {
     cy.visit(`/apply/${personId}/your-situation`);
   }
 
-  static getServedInArmedForcesRadioButton() {
-    const testId = 'test-radio-situation-armed-forces.1';
+  static getServedInArmedForcesRadioButton(index: number) {
+    const testId = `test-radio-situation-armed-forces.${index}`;
```
Comment (Duslerke) [inline cypress/pages/apply/[resident]/your-situation.ts:7]: Never seen a dot being used in a selector name.
Would guess it's not exactly conventional, but probably not an issue.

File: cypress/e2e/local/eligibleMainApplicantOnlyOver55.cy.ts [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+      //intercept address API since address is not relevant in this test
+      cy.intercept(
+        {
+          method: 'GET',
+          path: '/api/address/*',
+        },
+        addressSearchAPIResponse
+      ).as('addressSearchMock');
+
+      cy.viewport(1000, 1000);
```
Comment (Duslerke) [inline cypress/e2e/local/eligibleMainApplicantOnlyOver55.cy.ts:84]: How come a square view port?

**Review by Duslerke — APPROVED** [review]

File: cypress/pages/apply/[resident]/your-situation.ts (hunk 1 above)
Comment (LBHTKarki) [reply cypress/pages/apply/[resident]/your-situation.ts:7]: These controls use index based numbering in the id with `.`, so these test IDs follow the same convention. I agree that it's a bit unusual, but we can tidy those up later.

File: cypress/e2e/local/eligibleMainApplicantOnlyOver55.cy.ts (hunk 2 above)
Comment (LBHTKarki) [reply cypress/e2e/local/eligibleMainApplicantOnlyOver55.cy.ts]: It's just a random number to bump up the default viewport size a bit when running these manually in the GUI. I've updated it to 720p to make it a bit more standard.

**Review by sam-drumm — APPROVED** [review]

**Pull request merged**